src/jsn.erl

```diff
@@ -172,6 +173,36 @@ find(Path, Subpath, SearchTerm, Object) ->
     find(Subpath, SearchTerm, get(Path, Object)).
 
 
+-spec select(selection()|selections(), json_array()) -> json_array().
+%%------------------------------------------------------------------------------
+%% @doc Transforms a list of Elements according to a given Selectionification
+%%------------------------------------------------------------------------------
+select(Selection, Elements) when is_list(Elements) ->
```

Comment:
I think you can take the guard (and second clause) off here actually, and let this all fall down to `select/3`, personally.